Tests/Unit/MSFT_xFirewall.Tests.ps1

```diff
@@ -38,6 +38,9 @@ Import-Module -Name $DSCResourceModuleFile.FullName -Force
 $breakvar = $True
```

Comment:
Would you do me the favor of removal all of these, I accidently checked them in, I use them to break into scripts and forgot to clean up after myself. :grimacing: